backend/tests/conftest.py

```
@@ -32,79 +37,235 @@
+
+        # Try restarting the services
+        restart_result = subprocess.run(
+            ["docker-compose", "restart"], capture_output=True, text=True, check=False
+        )
+
+        if restart_result.returncode != 0:
+            logger.error(f"Failed to restart services: {restart_result.stderr}")
+            pytest.fail(f"Could not restart services: {restart_result.stderr}")
+
+        # Give services time to restart and check again
+        time.sleep(10)
```

Ruff (0.8.2)

Undefined name `time`

(F821)